### PR TITLE
added method to get artifacts file

### PR DESCRIPTION
**Pull request by Seretos**
Hihi, thanks for this api...

i have added a little method to get the artifacts zip file from a single build

### CODE REVIEW

**Comment by m4tthumphrey** [comment]
Can you remove the .idea from the gitignore please and fix the CS on the line I've commented on above. Hint: Add .idea to your [global ignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

File: test/Gitlab/Tests/Api/ProjectsTest.php [hunk 1]
```diff
@@ -292,6 +292,22 @@ public function shouldGetBuild()
...
+        $expectedArray = array();
+
+        $api = $this->getApiMock();
+        $api->expects($this->once())
+            ->method('get')
+            ->with('projects/1/builds/2/artifacts')
+            ->will($this->returnValue($expectedArray))
+        ;
+
+        $this->assertEquals($expectedArray,$api->buildArtifacts(1,2));
```
Comment (m4tthumphrey) [inline test/Gitlab/Tests/Api/ProjectsTest.php]: space between comma and 2

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,2 @@
 vendor/
-composer.lock
-.idea/
\ No newline at end of file
+composer.lock
```
Comment (m4tthumphrey) [inline .gitignore]: 🤔

**Comment by moufmouf** [comment]
@Seretos I think @m4tthumphrey is waiting for you to remove the addition of "composer.lock" in the `.gitignore` file before merging. Would you mind removing this? (I badly need this to be merged for one of my libs :) )

Thanks by the way to both of you for your great work!

**Comment by radutopala** [comment]
@Seretos can't seem to find the API endpoints for `/projects/:id/builds/:id/*` .. were those moved to https://docs.gitlab.com/ce/api/ci/builds.html?

**Comment by moufmouf** [comment]
@radutopala You don't want to use the Builds API you are referring to (https://docs.gitlab.com/ce/api/ci/builds.html). This is reserved to be used only by the runners.

You really want to use the Jobs API: https://docs.gitlab.com/ce/api/jobs.html#download-the-artifacts-file

By the way, I tested this PR in both Gitlab 8 and Gitlab 9 and it works.

**Comment by fbourigault** [comment]
I'm closing this PR as it use runners reserved API and it's superseded by #196. Thank you anyway!

**Pull request closed without merging**